protocol/messenger_share_urls_test.go

```diff
@@ -201,10 +201,16 @@ func (s *MessengerShareUrlsSuite) TestShareCommunityURLWithData() {
 func (s *MessengerShareUrlsSuite) TestParseCommunityURLWithData() {
 	community := s.createCommunity()
 
-	communityData, signature, err := s.m.prepareEncodedCommunityData(community)
-	s.Require().NoError(err)
-
-	url := fmt.Sprintf("%s/c/%s#%s", baseShareURL, communityData, signature)
+	// communityData, signature, err := s.m.prepareEncodedCommunityData(community)
+	// s.Require().NoError(err)
+
+	// https://status.app/c/GzAAAETnNgpsRpPBwMzNmQiokxNsTUoRiCDpdc4UDkxk1F7pOu65DV_H8-xat9EQ9g==#zQ3shr9nw5uqkXwZkFXBhSnsqifnj2QKXy4BgbdXrE1RhCSAV
+	// url := fmt.Sprintf("%s/c/%s#%s", baseShareURL, communityData, signature)
+	// note see https://github.com/status-im/status-web/pull/461/files#diff-378ed45ccbd1b990783b9cc2f2ee73430328530f17339176b4fa658f2b6eda08 for source
+	url := "https://status.app/c/iyKACkQKB0Rvb2RsZXMSJ0NvbG9yaW5nIHRoZSB3b3JsZCB3aXRoIGpveSDigKIg4bSXIOKAohiYohsiByMxMzFEMkYqAwEhMwM=#zQ3shYSHp7GoiXaauJMnDcjwU2yNjdzpXLosAWapPS4CFxc11"
```

Comment:
I suggest to put parsing of ready urls into separate test cases

Reply:
this is just a tmp commit to demo the fix

Reply:
ditto https://github.com/status-im/status-go/pull/3855#discussion_r1285708046